modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/token-definition.json

```diff
@@ -0,0 +1,473 @@
+{
+	"tokenCategories": [
+		{
+			"label": "general",
+			"name": "general"
+		},
+		{
+			"label": "colors",
+			"name": "colors"
+		},
+		{
+			"label": "typography",
+			"name": "typography"
+		},
+		{
+			"label": "spaces",
+			"name": "spaces"
+		}
+	],
+	"tokenSets": [
+		{
+			"label": "general",
+			"name": "general"
+		},
+		{
+			"label": "utility",
+			"name": "utility"
+		},
+		{
+			"label": "layout",
+			"name": "layout"
+		},
+		{
+			"label": "buttons",
+			"name": "buttons"
+		},
+		{
+			"label": "alerts",
+			"name": "alerts"
+		},
+		{
+			"label": "cards",
+			"name": "cards"
+		},
+		{
+			"label": "forms",
+			"name": "forms"
+		},
+		{
+			"label": "nav",
+			"name": "nav"
+		},
+		{
+			"label": "portlet",
+			"name": "portlet"
+		},
+		{
+			"label": "tables",
+			"name": "tables"
+		},
+		{
+			"label": "lists",
+			"name": "lists"
+		}
+	],
+	"tokens": [
+		{
+			"cssVariable": "white",
```

Comment:
@eduardoallegrini @izaera 
The mappings to a cssVariable need to adapted to:
```
{
    "editorType": "ColorPicker",
    "label": "gray-100",
     "mappings": [
          {
               "type": "cssVariable",
               "value": "gray-100"   
        }
    ],
    "name": "gray100Color",
    "tokenCategoryName": "colors",
    "tokenSetName": "general",
    "type": "String"
}
```
as discussed [here](https://github.com/liferay-frontend/liferay-portal/pull/119#discussion_r455074180) so the token-definition.json file conforms with the updated JSON schema

Reply:
ok! 😁

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/128